docs/contribute/contributing.md

```diff
@@ -116,14 +116,13 @@ Style:
 
     Example: _myHost_
 
-- If wrap using angle brackets `<>` within code examples, italic font is not supported.
+- If wrap using angle brackets <> within code examples, italic font is not supported.
```

Comment:
Hey, can you revert this change?

If we will are writing plain `<>`. It will most probably throw a Compilation Error saying "Expected corresponding JSX closing tag for <>". 

It tries to render `<>` as a component, instead of a text & any component without closing bracket causes an Error.